README.md

```diff
@@ -55,20 +55,50 @@ Documentation for ert is located at [https://ert.readthedocs.io/en/latest/](http
 ## Developing
 
 *ERT* is Python and C software. To start developing, install it in editable
-mode:
+mode. We suggest using [virtual
+environment](https://docs.python.org/3/library/venv.html) to isolate the
+install (substitute the appropriate way of sourcing venv for your shell):
 
 ```
 $ git clone https://github.com/equinor/ert
 $ cd ert
+$ python -m venv venv
```

Comment:
(I prefer not to have my venv inside the ert directory, as it will be wiped if I do a `git clean`)

Reply:
Sorry, total venv newb (tried to avoid dependency management in python in the past, as we only used python for one single project) - could you kindly make a suggestion for a well-suited location that could be used in the readme?